### PR TITLE
Added a more detailed description for the security hardening calendar

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -1,10 +1,21 @@
-#+title: Inventory users that have sudo access
+The =sudo= program allows a user to execute a command with elevated privileges.
+It is a good idea to limit and monitor who has access to do this.
+This module will check which users on a system can use sudo (=sudoers=) and report that as part of CFEngine Enterprise Inventory.
+In Mission Portal, the CFEngine Enterprise UI, you can get reports of all users with sudo access across all machines in your infrastructure:
 
-* Synopsis
+[[https://raw.githubusercontent.com/nickanderson/cfengine-inventory-sudoers/main/inventory-screenshot.png][[[https://raw.githubusercontent.com/nickanderson/cfengine-inventory-sudoers/main/inventory-screenshot.png]]]]
```
Comment (nickanderson) [inline README.org:6]: This is horrible. The image won't load in org-mode.

File: README.org [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
 
-- *Name*: =inventory-sudoers=
-- *Version*: =1.0.2=
-- *Description*: Inventory users that have sudo access from a set of users.
+(To see this, go to Reports -> Inventory -> Click the 3 dots at the top of a column -> Insert Column -> Users with sudo)
+
+You can also see the list of sudoers for a single machine from the command line:
+
+#+begin_src cf3
```
Comment (nickanderson) [inline README.org]: This should be an example block:

```
#+begin_example
$ sudo cf-agent --show-evaluated-vars -KI | grep "_inventory_sudoers.users"
nickanderson:_inventory_sudoers.users     {"root","ubuntu"}                                           source=promise
#+end_example
```

File: README.org [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+$ sudo cf-agent --show-evaluated-vars -KI | grep "_inventory_sudoers.users"
+nickanderson:_inventory_sudoers.users     {"root","ubuntu"}                                           source=promise
+#+end_src
+
+*Recommendation:* Review this list on a regular basis, and ensure all of the users really need =sudo= access.
+Remove =sudo= access where possible by [[editing the =/etc/sudoers= file][https://www.digitalocean.com/community/tutorials/how-to-edit-the-sudoers-file]].
```
Comment (nickanderson) [inline README.org]: org links have the target first.

```
 [[https://www.digitalocean.com/community/tutorials/how-to-edit-the-sudoers-file][editing the =/etc/sudoers= file]].
```

**Review by nickanderson — CHANGES_REQUESTED** [review]

File: README.org (hunk 1 above)
Comment (olehermanse) [reply README.org:6]: I don't have a better way currently.

Comment (nickanderson) [reply README.org:6]: Also, looks like some extra `[]` in there.

Should be `[[http://target][anchor text]]`

File: README.org (hunk 3 above)
Comment (olehermanse) [reply README.org]: Thanks, fixed.

File: README.org (hunk 2 above)
Comment (olehermanse) [reply README.org]: Fixed.

File: README.org (hunk 1 above)
Comment (olehermanse) [reply README.org:6]: @nickanderson it's supposed to be an image that is a link to itself.

Comment (nickanderson) [reply README.org:6]: HATE

**Review by nickanderson — APPROVED** [review]

**Pull request merged**